### PR TITLE
FEAT: bump version 0.2.dev0

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Bump version 0.2.dev0

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ jobs:
           twine upload --skip-existing ./**/*.tar.gz
         env:
           TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+          TWINE_PASSWORD: ${{ secrets.TWINE_TOKEN }}
```
Comment (MaxJPRey) [inline .github/workflows/ci.yml]:
```suggestion
          TWINE_PASSWORD: ${{ secrets.SYSTEMCOUPLING_TOKEN }}
```

Comment (MaxJPRey) [reply .github/workflows/ci.yml]: Fixed.

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Review by iboyd-ansys — APPROVED** [review]
Thanks @Revathyvenugopal162

**Pull request merged**